### ISSUE TITLE
titan doesn't lock on laser

### ISSUE THREAD

**Issue by kripto202** [acemod/ACE3#5013]
**Arma 3 Version: stable
**CBA Version: stable
**ACE3 Version: stable

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
titan compact launcher doesn't lock on laser designated targets. 

**Steps to reproduce:**
Happens on any type of laser, place a UAV laser designator and lase anywhere. Then attempt to lock on to the laser

**Where did the issue occur?**
everywhere

**Placed Modules:**
none

**Comment by BaerMitUmlaut** [comment]
It's an IR lock, not a laser lock. This is intended afaik.

**Comment by kripto202** [comment]
It was working before. Doesn't any locking system can lock on laser lock?

**Comment by BaerMitUmlaut** [comment]
No. The Javelin locks onto heat signatures as far as I know. Those are in a different part of the spectrum.

**Comment by kripto202** [comment]
not sure but this might help determine if it does or not https://www.reddit.com/r/explainlikeimfive/comments/2gbjjx/eli5_how_do_lockon_weapons_like_the_javelin_and/

**Comment by Drofseh** [comment]
The Titan is based on the real life Rafael Mini-Spike, which locks on using an infra-red sensor, not a laser.

**Comment by PabstMirror** [comment]
In vanilla can the titan lock on laser targets?

Laser designators sends short pulses in a very specific wavelength
I doubt javelin could pick it up.

**Comment by kripto202** [comment]
the titan does lock on to laser targets in vanilla

**Comment by Tavish101** [comment]
+1 for Titan locking on to "designated" targets, like you can do in vanilla.

Perhaps an alternative mode on the Titan Compact launcher that can lock on to laser targets?
Or an alternative mode on the Laser Designator that allows for Titans to target lock? 

It creates great gameplay opportunities like (with top-down attack) allowing for precision strikes when there is no arty, mortars or CAS available.

**Comment by Tavish101** [comment]
The British Army bought some of the newer Spike NLOS "Non Line Of Sight" variants to use for that exact purpose. The British designation is EXACTOR. The newer NLOS missiles can lock on to a laser using SAL. 

Quotes from the web:
> Semi-Active Laser (SAL) guidance techniques combine very high precision, with "man-in-the-loop" capability. The missile uses SAL to detect a coded laser spot which is created by a target designator illuminating the target. Ground designation, airborne designation and UAV designation provide effective support for laser targeting.

> Rafael is working on expanding the Spike NLOS missile's versatility by enhancing the existing EO-IR/CCD seeker with **semi-active laser (SAL)** capability and different anti-armour, blast-penetration, and high-explosive fragmentation warheads to meet specific applications.

**Comment by thirald** [comment]
I also encounter the same concern with the Titan AC or AA with Ace I even created a topic:/with the vanilla version everything works perfectly and nothing since with Ace in MoD

As you could see the screenshot with Ace 3.10.1:

http://image.noelshack.com/fichiers/2017/26/4/1498743114-20170629145557-1.jpg

And without the Ace 3.10.1:

http://image.noelshack.com/fichiers/2017/26/4/1498743114-20170629114504-1.jpg

For the record, I'm not the only one has encountered the problem since passing 64 bit wit

https://github.com/acemod/ACE3/issues/5319

**Comment by BaerMitUmlaut** [comment]
I have no idea what those screenshots are supposed to show nor how they are related to this issue.

**Comment by john681611** [comment]
Lack of education on Ace Titan use seems to have been dealt with in #5319

**Comment by Mike-MF** [comment]
Closing due to age, inactivity or already solved. May be re-opened if the issue is still present, even better just open a new up-to-date Issue.
